File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        lib = {}
        res= [0,0]
        for i in range(len(nums)):
            if((target - nums[i]) in lib):
                res[0] = lib[target-nums[i]]
                res[1]=i
                return res
            else:
                lib[nums[i]] = i
        print(lib)
        

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int i=0,sum=0,j=numbers.length-1;
        int res[] = new int[2];
        while(i<j){
            sum = numbers[i]+numbers[j];
            if(sum == target){ res[0]=i+1;res[1]=j+1;return res;}
            else if(sum < target) i++;
            else j--;
        }
        return res;
    }
}